README.md

```diff
@@ -4,22 +4,24 @@
 module "jinad" {
    source         = "jina-ai/jinad-aws/jina"
    version        = "0.0.5"
-   ebs = {
-     device_name = "/dev/sdh"
-     device_name_renamed = "/dev/xvdh"
-     mount_location = "/mnt/data"
-     jina_home = "/usr/local/jina"
-     }
    instances      = {
-     "encoder": {
-       "type": "c5.4xlarge",
-       "pip": [ "tensorflow>=2.0", "transformers>=2.6.0" ],
-       "command": "sudo apt install -y jq"
+     encoder: {
```

Comment:
Changes for readme should be in https://github.com/jina-ai/terraform-jina-jinad-aws/blob/main/main.tf#L6. Please avoid changing readme directly

Reply:
sure, I update the doc automatically by
`docker run -v $PWD:/module quay.io/terraform-docs/terraform-docs:0.11.2 markdown table /module > README.md`

Reply:
as mean as that,
before this PR, I have updated https://github.com/jina-ai/terraform-jina-jinad-aws/blob/main/main.tf#L6.
but forgot to update the readme by `docker run -v $PWD:/module quay.io/terraform-docs/terraform-docs:0.11.2 markdown table /module > README.md`

Reply:
Ah ok!